server/console_account.go

```diff
@@ -505,31 +505,3 @@ AND ((facebook_id IS NOT NULL
 
 	return &empty.Empty{}, nil
 }
-
-func checkWalletFormat(wallet map[string]interface{}, path string) error {
```

Comment:
@zyro We should discuss this change. I'll run it through with you before we merge this pull request.

Reply:
Sounds good. :+1: